From f30e8576ed48bfe302c6bcfaebe10db3fd072dd1 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Mon, 29 Aug 2011 18:28:53 +0800
Subject: ENGR00155464 mx6q: Change default DMA size to 184M

Change default DMA size to 184M for mx6q. Current 96M size
isn't enough for 1080P encoder + decoder, and HDMI output.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/configs/imx6_defconfig |   11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 6b0e6b7..7e7ec68 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/arm 2.6.38 Kernel Configuration
-# Tue Aug 16 14:53:37 2011
+# Mon Aug 29 18:28:10 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -255,8 +255,10 @@ CONFIG_ARCH_MXC=y
 # CONFIG_ARCH_OMAP is not set
 # CONFIG_PLAT_SPEAR is not set
 CONFIG_GPIO_PCA953X=y
+CONFIG_IMX_HAVE_PLATFORM_DMA=y
 CONFIG_IMX_HAVE_PLATFORM_FEC=y
 CONFIG_IMX_HAVE_PLATFORM_FSL_USB2_UDC=y
+CONFIG_IMX_HAVE_PLATFORM_GPMI_NFC=y
 CONFIG_IMX_HAVE_PLATFORM_IMX2_WDT=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_SNVS_RTC=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_I2C=y
@@ -291,7 +293,7 @@ CONFIG_IMX_HAVE_PLATFORM_IMX_ASRC=y
 # CONFIG_ARCH_MX5 is not set
 CONFIG_ARCH_MX6=y
 CONFIG_FORCE_MAX_ZONEORDER=13
-CONFIG_DMA_ZONE_SIZE=96
+CONFIG_DMA_ZONE_SIZE=184
 CONFIG_ARCH_MX6Q=y
 CONFIG_SOC_IMX6Q=y
 CONFIG_MACH_MX6Q_SABREAUTO=y
@@ -348,10 +350,12 @@ CONFIG_CPU_HAS_PMU=y
 # CONFIG_ARM_ERRATA_460075 is not set
 # CONFIG_ARM_ERRATA_742230 is not set
 # CONFIG_ARM_ERRATA_742231 is not set
+# CONFIG_PL310_ERRATA_588369 is not set
 # CONFIG_ARM_ERRATA_720789 is not set
 # CONFIG_ARM_ERRATA_743622 is not set
 # CONFIG_ARM_ERRATA_751472 is not set
 # CONFIG_ARM_ERRATA_753970 is not set
+# CONFIG_PL310_ERRATA_727915 is not set
 CONFIG_ARM_GIC=y
 
 #
@@ -579,7 +583,6 @@ CONFIG_WIRELESS=y
 CONFIG_WIRELESS_EXT=y
 CONFIG_WEXT_CORE=y
 CONFIG_WEXT_PROC=y
-CONFIG_WEXT_SPY=y
 CONFIG_WEXT_PRIV=y
 # CONFIG_CFG80211 is not set
 CONFIG_WIRELESS_EXT_SYSFS=y
@@ -692,6 +695,7 @@ CONFIG_MTD_NAND=y
 CONFIG_MTD_NAND_IDS=y
 # CONFIG_MTD_NAND_DISKONCHIP is not set
 # CONFIG_MTD_NAND_NANDSIM is not set
+# CONFIG_MTD_NAND_GPMI_NFC is not set
 # CONFIG_MTD_NAND_PLATFORM is not set
 # CONFIG_MTD_ALAUDA is not set
 # CONFIG_MTD_ONENAND is not set
@@ -1676,6 +1680,7 @@ CONFIG_DMADEVICES=y
 # CONFIG_MXC_PXP is not set
 # CONFIG_TIMB_DMA is not set
 CONFIG_IMX_SDMA=y
+# CONFIG_MXS_DMA is not set
 CONFIG_DMA_ENGINE=y
 
 #
-- 
1.7.9.5

